app/reducers/lnd.js

```diff
@@ -171,6 +176,12 @@ export function setStartLndError(errors) {
   }
 }
 
+export function clearStartLndError() {
```

Comment:
Style comment. Other action creators use arrow notation here. So this can be shortened to 
`export const clearStartLndError = () => ({
    type: CLEAR_START_LND_ERROR
  })`